portal-bridge/src/gossip.rs

```diff
@@ -173,6 +173,79 @@ async fn history_trace_gossip(
     })
 }
 
+/// Gossip any given content key / value to the state network.
+pub async fn gossip_state_content(
+    portal_clients: &Vec<HttpClient>,
```

Comment:
I'm a big fan of removing our support for multiple portal clients. Imo, it's confusing, adds unnecessary complexity for a feature nobody's using, exposes us to weird side affects when running multiple clients, the logs get all jumbled so it's hard to assess what's really going on, and iiuc, for state content, we want bridges to only gossip material that's addressed "near" to them. Out of bounds for this pr, but imo it's something worth doing. Maybe we can talk about it in sync tm

Reply:
I won't be able to attend meeting tomorrow as I mentioned on Monday, but I agree with you that sounds like the right direction